sshpiperd/utils/common.go

```diff
@@ -0,0 +1,17 @@
+package utils
+
+import (
+	"fmt"
+	"net"
+	"strings"
+)
+
+func FormatIPAddress(host string) string {
+	ip4 := net.ParseIP(host).To4()
+
+	if ip4 != nil || !strings.Contains(host, ":") {
+		return host
+	}
+
+	return fmt.Sprintf("[%s]", host)
```

Comment:
do you think we need to verify if it is IPv6 here?

Reply:
It's already verified above against ipv4 and `:` or what do you mean? :)

Reply:
`FormatIPAddress("127.0.0.1:22")` === `[127.0.0.1:22]`
is it correct?

Reply:
```
addr := fmt.Sprintf("%v:%v", utils.FormatIPAddress(host), port)
```
and
```
Host:             utils.FormatIPAddress(host),
Port:             port,
```
Is it possible to have `<host:port, port>` pair in the code as in above examples?

Reply:
ah you are right 🤣

Reply:
if it's possible, then yes, it needs to be fixed :smile: